docs/source/hpc-intro-text.rst

```diff
@@ -0,0 +1,23 @@
+**Definition:** HPC-stack is a repository that provides a unified, shell script-based build system for 
+building the software stack required for the `Unified Forecast System (UFS) <https://ufscommunity.org/>`_ and applications. 
+
+Background
+------------------------
+The UFS Weather Model draws on over 50 code libraries to run its applications. These libraries range from libraries developed in-house at NOAA (e.g. NCEPLIBS, FMS, etc.) to libraries developed by NOAA's partners (e.g. PIO, ESMF etc) to truly third party libraries (e.g. NETCDF). Individual installation of these libraries is not practical, so the `HPC-Stack <https://github.com/NOAA-EMC/hpc-stack>`_ was developed as a central installation system to ensure that the infrastructure environment across multiple platforms is as similar as possible. Installation of the HPC-Stack is required to run the SRW. 
+
+Instructions
+-------------------------
+`Level 1 <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_ platforms (e.g. Cheyenne, Hera) already have the HPC-Stack installed. Users on those platforms do *not* need to install the HPC-Stack before building UFS applications (e.g. SRW, MRW) or models. Users working on systems that fall under `Support Levels 2-4 <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`_ will need to install the HPC-Stack the first time they try to run UFS applications or models.
+
+Users can either build the HPC-stack on their local system or use the centrally maintained stacks on each HPC platform. For a detailed description of installation options, see :ref:`Installing the HPC-Stack <InstallBuildHPCstack>`.  
+
+.. note::
+   `HPC-Stack <https://github.com/NOAA-EMC/hpc-stack.git>`_ is part of the NCEPLIBS project and was originally written for the `Joint Effort for Data assimilation Integration (JEDI) <https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/>`_ framework.
```

Comment:
This is incorrect.
It was originally written by the developer for installing third-party libraries in a systematic manner on a `macOS` and `RHEL`.  It was later transferred, expanded and further enhanced in the JEDI project.

Reply:
@aerorahul regarding rtd (readthedocs) sphinx auto doc build option, rtd contains a direct webhook to hpc-stack repo. So irtd is supposed to automatically pick up the latest update from the git. Right now, @gspetro-NOAA is checking the doc with her forked branch. Once this PR is merged to develop branch, @gspetro-NOAA can update the webhook accordingly. So, I think we don't need to put a hook in gti action. @gspetro-NOAA can you put the rtd link on readme? So people can catch from readme as well.

Reply:
https://hpc-stack.readthedocs.io/en/latest/
Sorry for the delay--missed that last sentence when I originally read it. ;)